lang/en/logstore_xapi.php

```diff
@@ -121,3 +121,8 @@
 $string['failed'] = 'Failed';
 $string['resendevents'] = 'Resend ({$a->count})';
 $string['replayevent'] = 'Replay event';
+$string['confirmresendeventsheader'] = 'Set to replay events';
+$string['confirmresendevents'] = 'Warning: You are about send events to be replayed.
```

Comment:
I think the message needs to be shorter
https://uxplanet.org/5-essential-ux-rules-for-dialog-design-4de258c22116

What do you all think of something like:
```
You are about to send {$a->count} record(s) to the queue for reprocessing.
Do you wish to continue?
```
@Patches- @mlynn-lp @lzabo ?

Reply:
I agree.

Reply:
Could the "Do you wish to continue?" part of the message be shown on a new line?

Reply:
Nope, I can not find any exasmple how I can do that.
Just let me know how.